### PR TITLE
Fixing up the POM

### PR DESCRIPTION
**Pull request by jglick**
@reviewbybees

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -40,28 +38,21 @@
   </developers>
 
   <scm>
-    <connection>scm:git:git://github.com/jenkinsci/dashboard-view-plugin.git</connection>
-    <developerConnection>scm:git:git@github.com:jenkinsci/dashboard-view-plugin.git</developerConnection>
-    <url>https://github.com/jenkinsci/dashboard-view-plugin</url>
-    <tag>dashboard-view-2.9.8</tag>
```
Comment (jglick) [inline pom.xml]: A non-`HEAD` `tag` in `master` indicates some kind of mistake using the release plugin, possibly just using an obsolete version.

File: pom.xml [hunk 2]
```diff
@@ -74,40 +65,8 @@
...
-      <groupId>com.google.code.findbugs</groupId>
-      <artifactId>annotations</artifactId>
-      <version>3.0.0</version>
-      <scope>provided</scope>
-    </dependency>
-    <dependency>
-      <groupId>org.jenkins-ci.main</groupId>
-      <artifactId>jenkins-test-harness</artifactId>
-      <version>${jenkins.version}</version>
-      <scope>test</scope>
```
Comment (jglick) [inline pom.xml]: implied by parent

File: pom.xml [hunk 3]
```diff
@@ -74,40 +65,8 @@
...
-        <artifactId>findbugs-maven-plugin</artifactId>
-        <version>3.0.1</version>
-        <configuration>
-          <xmlOutput>true</xmlOutput>
-          <findbugsXmlWithMessages>true</findbugsXmlWithMessages>
-          <failOnError>false</failOnError>
-        </configuration>
-      </plugin>
-    </plugins>
-  </build>
```
Comment (jglick) [inline pom.xml]: implied by parent (other than `failOnError` setting)

File: pom.xml [hunk 4]
```diff
@@ -74,40 +65,8 @@
...
-      <artifactId>annotations</artifactId>
-      <version>3.0.0</version>
-      <scope>provided</scope>
-    </dependency>
-    <dependency>
-      <groupId>org.jenkins-ci.main</groupId>
-      <artifactId>jenkins-test-harness</artifactId>
-      <version>${jenkins.version}</version>
-      <scope>test</scope>
+      <version>2.10</version> <!-- last compatible with 1.554.x -->
```
Comment (jglick) [inline pom.xml]: `maven-plugin` was long since split out of core, you must pick a concrete version to depend on.

File: pom.xml [hunk 5]
```diff
@@ -5,7 +5,7 @@
   <parent>
     <groupId>org.jenkins-ci.plugins</groupId>
     <artifactId>plugin</artifactId>
-    <version>2.3</version>
+    <version>2.14</version>
```
Comment (jglick) [inline pom.xml]: 2.3 does not even work to cut releases any more.

File: pom.xml [hunk 6]
```diff
@@ -40,28 +38,21 @@
...
-      <id>maven.jenkins-ci.org</id>
-      <url>http://maven.jenkins-ci.org:8081/content/repositories/releases/</url>
-    </repository>
-  </distributionManagement>
 
   <repositories>
     <repository>
       <id>repo.jenkins-ci.org</id>
-      <url>http://repo.jenkins-ci.org/public/</url>
+      <url>https://repo.jenkins-ci.org/public/</url>
```
Comment (jglick) [inline pom.xml]: `http` now rejected.

File: pom.xml [hunk 7]
```diff
@@ -40,28 +38,21 @@
...
+    <connection>scm:git:git://github.com/jenkinsci/${project.artifactId}-plugin.git</connection>
+    <developerConnection>scm:git:git@github.com:jenkinsci/${project.artifactId}-plugin.git</developerConnection>
+    <url>https://github.com/jenkinsci/${project.artifactId}-plugin</url>
   </scm>
-  <distributionManagement>
-    <repository>
-      <id>maven.jenkins-ci.org</id>
-      <url>http://maven.jenkins-ci.org:8081/content/repositories/releases/</url>
-    </repository>
-  </distributionManagement>
```
Comment (jglick) [inline pom.xml]: implied by parent

File: pom.xml [hunk 8]
```diff
@@ -40,28 +38,21 @@
   </developers>
 
   <scm>
-    <connection>scm:git:git://github.com/jenkinsci/dashboard-view-plugin.git</connection>
-    <developerConnection>scm:git:git@github.com:jenkinsci/dashboard-view-plugin.git</developerConnection>
-    <url>https://github.com/jenkinsci/dashboard-view-plugin</url>
-    <tag>dashboard-view-2.9.8</tag>
+    <connection>scm:git:git://github.com/jenkinsci/${project.artifactId}-plugin.git</connection>
```
Comment (jglick) [inline pom.xml]: standard pattern

File: pom.xml [hunk 9]
```diff
@@ -18,9 +18,7 @@
 
   <properties>
     <jenkins.version>1.554.1</jenkins.version>
-    <project.reporting.outputEncoding>UTF-8</project.reporting.outputEncoding>
-    <project.build.outputEncoding>UTF-8</project.build.outputEncoding>
-    <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
```
Comment (jglick) [inline pom.xml]: implied by parent

**Comment by stephenc** [comment]
:bee:

**Comment by stephenc** [comment]
@reviewbybees done

**Pull request merged**